src/components/BounceRulesContainer/index.jsx

```diff
@@ -80,7 +80,9 @@ const BounceRulesContainer = ({
     <Row>
       <Column width={6} offset={2}>
         <Breadcrumb>
-          <a href="/bounce_rules">Bounce Rules</a>
+          <a data-test="bounce-rules-page" href="/bounce_rules">
```

Comment:
Eventually in your components you would spread the writeselectors onto here like {...WriteSelectors.bounceRulesPage}